Formula/ttyd.rb

```diff
@@ -0,0 +1,25 @@
+class Ttyd < Formula
+  desc "Command-line tool for sharing terminal over the web"
+  homepage "https://github.com/tsl0922/ttyd"
+  url "https://github.com/tsl0922/ttyd/archive/1.0.0.tar.gz"
+  sha256 "6fae4e932280f0ca430ea834f88282673351a321535acbe3725e995e83857138"
+  head "https://github.com/tsl0922/ttyd.git"
+
+  depends_on "cmake" => :build
+  # xxd tool is from vim
```

Comment:
Not necessary, remove this comment.

However, isn't `/usr/bin/xxd` good enough?